src/reducers/workspaces.ts

```diff
@@ -301,7 +302,15 @@ export const reducer: Reducer<IWorkspaceManagerState> = (
         ...state,
         [location]: {
           ...state[location],
-          isRunning: true
+          isRunning: true,
+          editorTestcases: state[location].editorTestcases.map((testcase, i) => {
+            if (i === action.payload.testcaseId) {
+              testcase.actual = undefined;
```

Comment:
what is `actual` supposed to be? It is hard to tell from the name plus it seems to have `any` type?

Reply:
Refactored